app/collections/page.tsx

```diff
@@ -10,6 +16,22 @@ export const metadata: Metadata = {
   },
 };
 
-export default async function Collections() {
-  return <CollectionsPage data={mockCollectionCards} />;
+export default async function Collections({ searchParams }: CollectionsProps) {
+  const data = await getCollectionsData({
+    keyword: searchParams.collection_keyword,
+    sortID: searchParams.sort,
+    pageNum: searchParams.page,
```

Comment:
I'm assuming that `collection_keyword`, `sort`, and `page` are all param names needed for Repo API?

This isn't wrong but try this syntax:
```
const { collection_keyword: keyword, sort: sortID, page: pageNum } = searchParams;
  const data = await getCollectionsData({
    keyword,
    sortID,
    pageNum,
  });
```

This is to separate the naming convention from the URL query params and the naming convention you need in the app.

Reply:
Also just want to note here that in the OG DC url query, it's `collection_keywords`, not `collection_keyword`, if we're thinking about URL parity

Reply:
If we can build out a redirect service, I'd like `collection_keywords` to be `q`. That's the conventional query key search (for example https://www.google.com/search?q=cats).

Reply:
good find @7emansell thank you.

@EdwinGuzman I would like that as well. However, since we do not have a redirect service atm, `collection_keywords` will be the expected param.